.travis.yml

```diff
@@ -18,8 +18,13 @@ before_install:
 
 install:
   - pip install -e .
+  - cd dataset
+  - bash download.sh
```

Comment:
Because the test case doesn't train on this data, are we running this script to make sure that the data is available from Zenodo?  That would be a reasonable check.

Reply:
Exactly.